### PR TITLE
Prevent mnemonic screen capture & fix click on first import word

### PR DESCRIPTION
**Pull request by nop33**
Closes https://github.com/alephium/mobile-wallet/issues/251

### CODE REVIEW

**Comment by mvaivre** [comment]
> could you please check if this PR fixes the Face ID issue that you mentioned here?

It does! 🚀

File: src/components/inputs/Input.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const Input = <T extends InputValue>({
   const showCustomValueRendering = typeof renderedValue !== 'string' && renderedValue !== undefined
 
   const labelStyle = useAnimatedStyle(() => ({
-    top: withSpring(!isActive ? -10 : -40, fastSpringConfiguration)
+    bottom: withSpring(!isActive ? 0 : 30, fastSpringConfiguration)
```
Comment (mvaivre) [inline src/components/inputs/Input.tsx:74]: Good catch :)

**Review by mvaivre — APPROVED** [review]
👍

File: src/components/inputs/Input.tsx (hunk 1 above)
Comment (nop33) [reply src/components/inputs/Input.tsx:74]: a frustrating one! Thanks to the inspect mode

**Pull request merged**